apps/web/src/stores/documentLogs.ts

```diff
@@ -21,45 +19,12 @@ export default function useDocumentLogs(
   },
   { fallbackData }: SWRConfiguration = {},
 ) {
-  const { toast } = useToast()
-  const fetcher = useCallback(async () => {
-    try {
-      const response = await fetch(
-        // TODO: Move to ROUTES once implemented
-        `/api/projects/${projectId}/commits/${commitUuid}/documents/${documentUuid}/documentLogs?page=${page ?? undefined}&pageSize=${pageSize ?? undefined}`,
-        {
-          credentials: 'include',
-          headers: {
-            'Content-Type': 'application/json',
-          },
-        },
-      )
-
-      if (!response.ok) {
-        const error = await response.json()
-
-        toast({
-          title: 'Error',
-          description: error.message,
-          variant: 'destructive',
-        })
-
-        return []
-      }
-
-      return await response
-        .json()
-        .then((rows) => rows.map(documentLogPresenter))
-    } catch (error) {
-      toast({
-        title: 'Error',
-        description: (error as Error).message,
-        variant: 'destructive',
-      })
-
-      return []
-    }
-  }, [commitUuid, documentUuid, projectId, toast, page, pageSize])
+  const fetcher = useFetcher(
+    `/api/projects/${projectId}/commits/${commitUuid}/documents/${documentUuid}/documentLogs?page=${page ?? undefined}&pageSize=${pageSize ?? undefined}`,
```

Comment:
`useFetcher` could have a `searchParams` argument that accept an object and convert to string

Reply:
Also use ROUTES

Reply:
💯